File: Lab8/ModifiedMVCExample/SourceCode/Model/Student.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Model;


public class Student {
   private String name;
   private String regNo;
   private int semister;
   private double gpa;
   public Student(String name, String regNo, int semister, double gpa){
       this.name = name;
       this.semister = semister;
       this.regNo = regNo;
       this.gpa = gpa;
   }
   public void setName(String name){
       this.name = name;
   }
   public void setRegNo(String regNo){
       this.regNo = regNo;
   }
   public void setSemister(int semister){
       this.semister = semister;
   }
   public void gpa(double gpa){
       this.gpa = gpa;
   }
   
   public String getName(){
       return name;
   }
   public String getRegNo(){
       return regNo;
   }
   public int getSemister(){
       return semister;
   }
   public double getGpa(){
       return gpa;
   }
   
}

File: Lab8/MVCInPython/InSimplePython/model/student.py
# model/student.py

class Student:
    def __init__(self, name, reg_no, semester, gpa):
        self._name = name
        self._reg_no = reg_no
        self._semester = semester
        self._gpa = gpa

    # Setters
    def set_name(self, name):
        self._name = name

    def set_reg_no(self, reg_no):
        self._reg_no = reg_no

    def set_semester(self, semester):
        self._semester = semester

    def set_gpa(self, gpa):
        self._gpa = gpa

    # Getters
    def get_name(self):
        return self._name

    def get_reg_no(self):
        return self._reg_no

    def get_semester(self):
        return self._semester

    def get_gpa(self):
        return self._gpa
